--- frescobaldi_app/lilypondinfo.py.orig	2014-08-25 18:04:02.000000000 +0200
+++ frescobaldi_app/lilypondinfo.py	2014-08-25 18:06:28.000000000 +0200
@@ -298,7 +298,7 @@
                 toolpath += '.py'
             command = [self.python(), toolpath]
         else:
-            command = [toolpath]
+            command = [os.path.abspath('../MacOS/python'), toolpath]
         return command
     
     @CachedProperty.cachedproperty(depends=versionString)
