--- frescobaldi_app/portmidi/pm_ctypes.py.orig	2014-06-14 12:05:46.000000000 +0200
+++ frescobaldi_app/portmidi/pm_ctypes.py	2014-06-14 12:05:49.000000000 +0200
@@ -95,6 +95,7 @@
 else:
     from ctypes.util import find_library
     dll_name = find_library(_PM_DLL.get(sys.platform, 'portmidi'))
+    dll_name = '../Frameworks/libportmidi.dylib'
 if dll_name is None:
     raise ImportError("Couldn't find the PortMidi library.")
 
